lexicons/app/bsky/graph/getAssertions.json

```diff
@@ -71,20 +71,10 @@
       "properties": {
         "cid": {"type": "string"},
         "actorType": {
-          "oneOf": [
-            {"$ref": "#/defs/actorKnown"},
-            {"$ref": "#/defs/actorUnknown"}
-          ]
+          "type": "string", 
+          "knownValues": ["app.bsky.system.actorUser", "app.bsky.system.actorScene"]
```

Comment:
Still haven't totally wrapped my head around the `token` story— would it make sense to say that the `type` of this field is `token`?  Generally sort of wondering—how (if at all) do we tie together the existence of these tokens with the validation of this property?

Reply:
Ah, no the token type is just for declaring empty NSIDs, not for consuming tokens. But that's an easy misunderstanding and we might need to think about clarity.